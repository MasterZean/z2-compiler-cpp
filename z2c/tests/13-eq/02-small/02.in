-100s8 == Bool{-100}
-100s8 == -100s8
-100s8 == 156u8
-100s8 == -100s16
-100s8 == 65436u16
-100s8 == -100
-100s8 == 4294967196u
-100s8 == -100l
-100s8 == 18446744073709551516ul
-100s8 == -100.0f
-100s8 == -100.0
-100s8 == 4294967196p
