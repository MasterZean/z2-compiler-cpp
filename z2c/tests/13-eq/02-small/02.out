-100 == (bool)(-100);
-100 == -100;
-100 == 156u;
-100 == -100;
-100 == 65436u;
-100 == -100;
-100 == 4294967196u;
-100 == -100ll;
-100 == 18446744073709551516ull;
-100 == -100.0f;
-100 == -100.0;
-100 == 4294967196u;
